pnpm/src/cmd/recursive.ts

```diff
@@ -102,6 +102,10 @@ Convenient to use in a multi-package repository.',
 A shared lockfile also means that all dependencies of all projects will be in a single node_modules.`,
             name: '--shared-workspace-lockfile',
           },
+          {
+            description: 'When executing commands recursively in a workspace, execute them on the root workspace project as well.',
```

Comment:
When there is one sentence in help, we don't put dot in the end.

Reply:
okay